### PR TITLE
FE: Adjust FormField to be able to display checkbox with the label nice

### PR DESCRIPTION
**Pull request by uladzimirdev**
Extend FormField, so it will be able to show a checkbox

Example in storybook:
`/story/core-forminput--with-title-and-actions`

image

### CODE REVIEW

File: frontend/src/metabase/core/components/FormInput/FormInput.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ export interface FormInputProps
   > {
   name: string;
   title?: string;
+  titleActions?: ReactNode;
```
Comment (ranquild) [inline frontend/src/metabase/core/components/FormInput/FormInput.tsx:20]: maybe just `actions`? I've this property in other components in the app. I don't think we would add actions in another place; and semantically we probably shouldn't be thinking that actions are specifically added near the title

File: frontend/src/metabase/core/components/FormInput/FormInput.stories.tsx [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
 import Form from "../Form";
 import FormProvider from "../FormProvider";
+import CheckBox from "../CheckBox";
 import FormInput from "./FormInput";
 
+const TitleActions = () => {
+  const [checked, setChecked] = useState(true);
+  return (
+    <CheckBox
+      onChange={() => setChecked(checked => !checked)}
```
Comment (ranquild) [inline frontend/src/metabase/core/components/FormInput/FormInput.stories.tsx:12]: `onChange={setChecked}`

File: frontend/src/metabase/actions/components/ActionFormFieldWidget/ActionFormFieldWidget.tsx [hunk 3]
```diff
@@ -32,16 +33,24 @@ interface FormWidgetProps {
   formField: ActionFormFieldProps;
 }
 
-const ActionFormFieldWidget = forwardRef(function FormFieldWidget(
+export const ActionFormFieldWidget = forwardRef(function FormFieldWidget(
   { formField }: FormWidgetProps,
   ref: React.Ref<any>,
 ) {
+  const [hidden, setHidden] = useState(false);
```
Comment (ranquild) [inline frontend/src/metabase/actions/components/ActionFormFieldWidget/ActionFormFieldWidget.tsx:40]: Mapping this to the correct property will be done in another PR?

**Review by ranquild — APPROVED** [review]

File: frontend/src/metabase/core/components/FormInput/FormInput.tsx (hunk 1 above)
Comment (uladzimirdev) [reply frontend/src/metabase/core/components/FormInput/FormInput.tsx:20]: I'm fine with any name, your point makes sense to me 👍

File: frontend/src/metabase/actions/components/ActionFormFieldWidget/ActionFormFieldWidget.tsx (hunk 3 above)
Comment (uladzimirdev) [reply frontend/src/metabase/actions/components/ActionFormFieldWidget/ActionFormFieldWidget.tsx:40]: I pushed it by mistake, sorry
